web/packages/plugins/ui/sima-application-input/src/EditSimaApplicationInput.tsx

```diff
@@ -23,17 +26,47 @@ import {
 import { NotificationManager } from 'react-notifications'
 import {
   Column,
+  Container,
   EditInputEntity,
   GroupWrapper,
   HeaderWrapper,
   Row,
 } from './components'
 import _ from 'lodash'
 
+const ReadOnlyPathTextField = (props: { path: string }) => {
+  return (
+    <Tooltip
+      title={truncatePathString(props.path) === props.path ? '' : props.path}
+    >
+      <Input
+        variant={'default'}
+        style={{
+          width: PATH_INPUT_FIELD_WIDTH,
+          cursor: 'default',
+        }}
+        type="string"
+        value={truncatePathString(props.path)}
+      />
+    </Tooltip>
+  )
+}
+
+const ReadOnlyTextField = (props: { text: string; label: string }) => {
+  return (
+    <TextField
```

Comment:
TextFields have readOnly property. Maybe also use that to disable the possibility to write in the fields.